### DISCUSSION TITLE
why do the tall and wide CSV formats treat additional notes differently?

### DISCUSSION THREAD

**Discussion by nmaylevc** [CMSgov/hospital-price-transparency#131] [Q&A] [answered]
Why do the tall and the wide CSV formats treat the additional notes differently?

It is useful to separate out what is a generic note about the code, description, gross charge and other generic fields from the payer specific fields.  The current design of the tall format has eliminated the additional_generic_notes and seems to expect people to fold that information into the additional_payer_notes.

Two concerns with this:

* one could have folded the generic info into each of the payer specific notes fields in the wide format too. This idea isn't unique to tall CSV.  The only thing special about tall CSV is that the generic notes field would have been repeated but that is also true for many of the fields in the tall format -- repetition is the downside of a flat file format.  Why make the notes field special?
* ideally, the different formats would be equivalent in fields since they are just alternative representations of the same underlying data rather than different data.  By creating this difference the disclosures will be different based on the format chosen.  This also means that if users load this data into a database they will get different results depending on the file format.

The other issue is that this results in the description of the additional generic notes field being incorrect for the tall format since now this field is intended to have both generic and payer-plan specific notes combined in some way that won't be able to be easily detangled.  I see that this is covered by a "footnote", but that just shows that this is confusing enough to need clarification.

Can this be fixed?  

btw: we see files that have included both columns in tall CSV format.

**Comment by RidenourK** [comment, marked as answer]
@nmaylevc. Thank you for your question. 

In the CY 2022 OPPS/ASC proposed rule ([86 FR 42321](https://www.federalregister.gov/citation/86-FR-42321)), CMS sought comment on improving standardization of the data disclosed by hospitals in the MRF. In response, many commenters urged CMS to create a standard template for hospitals to use for posting their MRF, noting that such standardization could ease operational burdens, improve the public's (including employers and researchers) ability to make price comparisons across hospitals, and better enable third party data aggregation services to develop user-friendly patient tools for displaying this information. Some commenters recommended that CMS work with providers and vendors to better understand the benefits of a standard template. CMS requested that the HHS Health Federally Funded Research and Development Center (FFRDC) more fully explore the feasibility of these commenters' recommendations and identify technical specifications and categories of information (referred to as “data elements”) that we could consider proposing in future rulemaking to improve the usability and meaningfulness of the standard charges display. The FFRDC convened a technical expert panel (TEP) and used the TEP members' advice to make informed recommendations to CMS in the summer of 2022.

The TEP members discussed what general machine-readable format(s) would be best suited to display hospital standard charges, including CSV “tall”, CSV “wide”, and JSON. The TEP members suggested that hospitals have a choice of JSON and CSV formats. It was also suggested by innovator representatives that two notes fields should be created in the MRF: one for specific notes related to a particular payer specific standard charge displayed in the CSV wide format, and one for all notes related to the standard charges. Please refer to pages 2-3 and 3-13 of the [TEP Report](https://mitre.app.box.com/v/MITRE-MRF-TEP/file/1244177570809).

**Reply by nmaylevc** [reply]
The notes fields are a great addition to the file standard.  I am simply asking for consistency across the three standard templates.  Today the tall CSV template doesn't have both types of notes.

**Reply by angelina-abitino** [reply]
I agree. This makes it very confusing when trying to aggregate many provider files of different formats (Tall CSV, Wide CSV, and JSON). I think many would benefit from some updates to the schema that would make the representations equal across the three formats. It doesn't make sense to lose the additional_payer_notes field in the Tall format. 

Calling it additional_generic_notes in the Tall format is confusing if payer-specific info is expected to populate it. I see that knh762002 has come up with their own custom solutions (including general and payer info separated by a pipe (|) in the Tall CSV additional_generic_notes field), which I think makes sense, but if every provider implements their own custom solution, then we are moving backwards and away from standardization. These custom solutions will not allow for a streamlined aggregation of many provider files.

Will CMS be considering this feedback and make any updates before 2025?

**Comment by knh762002** [comment]
CMS has provided the following text in their documentation.  "Additional Generic Notes Notes:  If using the CSV Tall template, this data element can be used for both additional payer-specific and general information about the standard charge for an item or service. If using the CSV Wide template, use the ‘Additional Generic Notes’ data element for additional general information and use the ‘Additional Payer-Specific Notes’ data element for additional payer-specific information."  

From my experience...in the tall format, I will generally only have 1 payer/plan per set of code types per line.  So, any additional_payer_notes that you would have encoded in the wide format can just be encoded in the additional_generic_notes column in the tall format.  Should you have the need to also include notes that are not payer/plan specific, you can either encode those notes into the additional_generic_notes column along with your payer/plan notes or just add a new column to the tall format to include those notes.  I encode both general notes and payer/plan notes in the same additional_generic_notes by splitting them using a bar (|) character.  Again, this is just from my experience.

As for creating new columns in .CSV files or objects in the .JSON file to provide additional context, CMS has stated in other discussions that it is not prohibited.

https://github.com/CMSgov/hospital-price-transparency/discussions/73#discussioncomment-9191953

Hope this helps...

**Reply by nmaylevc** [reply]
Thank you for the suggestion.  That would be a good way to segregate the generic notes from the payer specific notes.  However, not everyone will follow your convention, so generally, users of the data will have to write code to heuristically separate the different types of notes from the one field.

Since it was deemed useful to have both types of notes, why mix them up in one of the templates?  The different templates should not be offering different fields, but simply different representations of the same underlying data.  As one of the users of this data, I should be indifferent to which template the payer chose; all templates should represent the same data. This difference in how the notes are encoded breaks that.  As does the difference between modifiers in JSON versus CSV.  It would be very good to reconcile all of those differences.

**Reply by angelina-abitino** [reply]
I agree. The more we begin adding new columns and our own rules, the more we move away from a standardized schema, and we're back to  the way things were before the schema requirements were in effect. There should be no differences between the Tall, Wide, and JSON formats. Another major inconsistency is the way modifiers are expected to be provided between the CSV and JSON formats. It's completely different.
